tests/pcmjava/tools.vitruv.applications.pcmjava.pojotransformations.java2pcm.tests/src/tools/vitruv/applications/pcmjava/tests/pojotransformations/java2pcm/InterfaceMappingTransformationTest.java

```diff
@@ -2,17 +2,25 @@
 
 import static org.junit.Assert.assertTrue;
 
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
 import org.eclipse.emf.ecore.EObject;
+import org.junit.Before;
 import org.junit.Test;
 
 import org.palladiosimulator.pcm.repository.BasicComponent;
 import org.palladiosimulator.pcm.repository.OperationInterface;
 import org.palladiosimulator.pcm.repository.Repository;
 
+import tools.vitruv.applications.pcmjava.pojotransformations.java2pcm.Java2PcmUserSelection;
 import tools.vitruv.applications.pcmjava.tests.util.Pcm2JavaTestUtils;
 
 public class InterfaceMappingTransformationTest extends Java2PcmPackageMappingTransformationTest {
 
```

Comment:
Logger should never be enabled by default. A VM argument for enabling debugging exists.